epam-promo/components/pickers/PickerInput.tsx

```diff
@@ -111,6 +117,7 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
                         maxHeight={ maxHeight }
                         searchSize={ this.props.size }
                         editMode='dropdown'
+                        renderNotFound={ this.props.renderNotFound ? () => this.props.renderNotFound() : this.renderDefaultNoFound }
```

Comment:
move it into separate function, because now we will have new prop on each render